src/components/table/partial-styles/tabulator-custom-styles.scss

```diff
@@ -1,32 +1,87 @@
 @use '../../../style/mixins';
 @use '../../../style/functions';
 
+.interactive-feedback {
+    // This is a div, injected by us into all rows.
+    // We use it to visualize interactive feedback.
+    position: absolute;
+    inset: 0;
+    min-height: 0.125rem;
+
+    // We could use `@include mixins.is-flat-clickable()`,
+    // But the "arguments" of the mixin would make the CSS for
+    // even and odd row colors and background color very complicated.
+    // So instead, I've copy/pasted the useful parts of the mixin here
+    transition: color 0.2s ease, background-color 0.2s ease,
+        box-shadow 0.2s ease, transform 0.1s ease-out;
+}
+
+.tabulator-row:not(.tabulator-calcs) {
+    .tabulator-cell {
+        // We don't want the cells to block
+        // the hovering possibility of `interactive-feedback` div.
+        pointer-events: none;
+        // but we want the content in them
+        // (e.g. links, phone numbers, chips, components)
+        // to remain interactive.
+        > * {
+            pointer-events: all;
```

Comment:
Should we use auto here?

Reply:
oh yes. good point!

Reply:
https://github.com/Lundalogik/lime-elements/assets/35954987/28ab243e-7e57-4c4f-be88-6a4b0208eaf5